.markdownlint.json

```diff
@@ -2,5 +2,6 @@
   "heading-style": { "style": "atx" },
   "ul-style": { "style": "dash" },
   "line-length": { "line_length": 120, "stern": true },
-  "hr-style": { "style": "---" }
+  "hr-style": { "style": "---" },
+  "MD013": false
```

Comment:
Maybe use this on the line where its needed instead?
```
<!--- markdownlint-disable MD013 --->
```

Reply:
Was this just for links? If so, use reference links instead - the line length isn't validated there and imo is cleaner anwyay.

Reply:
Fixed with formatting the link. 8547c66e37df72c03e1d25bc4a469e65bcde2972

Reply:
The commit is in this PR right now. Can you add it?

Reply:
Cherry-picked from #321